### PR TITLE
refactor: update manifest based on openAI plugin

### PR DESCRIPTION
**Pull request by yuqizhou77**
- update scaffolded manifest based on open AI plugin manifest

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -207,6 +207,8 @@
   "error.generator.DownloadSampleApiLimitError": "Unable to download sample due to throttling. Retry later after rate limit reset (This may take up to 1 hour). Alternatively, you can go to %s to git clone the repo manually",
   "error.generator.DownloadSampleNetworkError": "Unable to download sample due to network error. Check your network connection and retry. Alternatively, you can go to %s to git clone the repo manually",
   "error.generator.ParseUrlError": "Unable to parse url %s",
+  "error.copilotPlugin.openAiPluginManifest.CannotGetLogoError": "Unable to get logo from '%s'. Please update '%s' manually if you want to use same logo for your Copilot plugin.",
+  "error.copilotPlugin.openAiPluginManifest.CannotGetManifest": "Unable to get OpenAI plugin manifest from '%s'.",
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/package.nls.json:211]: Better for us not to convert the logo/icon as the Teams app has specific icon sizing requirements. We can follow practice for other templates where we use a default one and developers can change by themselves.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -207,6 +207,8 @@
...
   "error.m365.NotExtendedToM365Error": "Unable to extend Teams app to Microsoft 365. Use 'teamsApp/extendToM365' action to extend your Teams app to Microsoft 365.",
   "core.QuestionAppName.validation.pattern": "Application name must start with letters and contain at least two letters or digits. It can not contain some special characters.",
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/package.nls.json:213]: When would we pop up this error message?

Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json:213]: @kuojianlu could you please help on this - when will "error.m365.NotExtendedToM365Error" show to users?

Comment (kuojianlu) [reply packages/fx-core/resource/package.nls.json:212]: When users try preview a M365 app in Outlook/Office but the app has not been extended to M365.

**Review by sffamily — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**